src/services/rsvp/rsvp-router.test.ts

```diff
@@ -0,0 +1,112 @@
+import { describe, expect, it, beforeEach } from "@jest/globals";
+import { TESTER, getAsAttendee, getAsStaff, putAsApplicant, getAsAdmin } from "../../testTools.js";
+import { DecisionStatus, DecisionResponse } from "../../database/decision-db.js";
+import Models from "../../database/models.js";
+import { StatusCode } from "status-code-enum";
+
+const TESTER_DECISION_INFO = {
+    userId: TESTER.id,
+    status: DecisionStatus.ACCEPTED,
+    response: DecisionResponse.PENDING,
+    reviewer: "reviewer1",
+    emailSent: true,
+};
```

Comment:
Is this absolutely necessary? I made it so that the DecisionInfo model extends Document, since I needed to use .toObject() from the MongoDB library so adding "satisfies DecisionInfo" makes the type checker mad.

Reply:
None of the other routers have needed to extend Document so far, I'm not sure why you need to? Having some kind of check to verify our tests are testing the right things is important.